src/go/rdctl/pkg/autostart/autostart_linux.go

```diff
@@ -133,14 +136,33 @@ func findApplicationFilePath() (string, error) {
 // Gathers the info that is needed to fill out the autostart .desktop
 // file template.
 func getAutostartFileData() (autostartFileData, error) {
-	// TODO: update this code once paths tracking is moved into rdctl
-	rancherDesktopSymlinkPath := filepath.Join(xdg.Home, ".rd", "bin", "rancher-desktop")
-	appImagePath, err := filepath.EvalSymlinks(rancherDesktopSymlinkPath)
-	if err != nil {
-		return autostartFileData{}, fmt.Errorf("failed to resolve %q: %w", rancherDesktopSymlinkPath, err)
+	executablePath := ""
+	if _, ok := os.LookupEnv("APPIMAGE"); ok {
+		// If we're running under AppImage, then we need to look up
+		// ~/.rd/bin/rancher-desktop and resolve it to find the executable path.
+		// We only expect to be run from the UI here, so the environment
+		// variable should be set correctly.
+		paths, err := p.GetPaths()
```

Comment:
We have settled on importing the package as `paths` and using `appPaths` as the local variable name in almost all places:

```suggestion
		appPaths, err := paths.GetPaths()
```

But there are a few instances with the old style left, so we can clean this up later all together.